packages/mwp-core/src/renderers/server-render.jsx

```diff
@@ -277,26 +277,30 @@ const makeRenderer = (
 		};
 		const checkReady = state =>
 			state.preRenderChecklist.every(isReady => isReady);
-		const populateStore = store =>
-			new Promise((resolve, reject) => {
+		const populateStore = store => {
+			return addFlags(store).then(() => {
```

Comment:
nitpick, skip the curly brackets and `return`

Reply:
made this change but then forgot to push 😢